### PR TITLE
complete task 9

### PR DESCRIPTION
**Pull request by winniekwoknorth**

### CODE REVIEW

File: controllers/comments.controllers.js [hunk 1]
```diff
@@ -19,9 +19,19 @@ exports.postCommentsByArticleId = (req, res, next) => {
...
-        console.log(comments)
         res.status(201).send({comments})
     })
     .catch(next)
 }
 
+exports.deleteCommentByCommentId = (req, res, next) => {
+    const { comment_id } = req.params
+    const commentPromises = [deleteComments(comment_id)]
+    if(comment_id){commentPromises.push(checkCommentIdExists(comment_id))}
```
Comment (aloush-dev) [inline controllers/comments.controllers.js:30]: this if statement isn't really needed.

**Review by aloush-dev — APPROVED** [review]
Awesome job.
We don't really need to check if the comment exists because we should get a PSQL error.

Merge when ready.
Thanks

**Review by aloush-dev — APPROVED** [review]
Awesome job.
We don't really need to check if the comment exists because we should get a PSQL error.

Merge when ready.
Thanks

File: controllers/comments.controllers.js (hunk 1 above)
Comment (winniekwoknorth) [reply controllers/comments.controllers.js:30]: I seem to still get a 204 response with comment_id =99 without the before adding commentid check

**Pull request merged**